go/labels/warehouse.go

```diff
@@ -1,9 +1,12 @@
-
+// Package labels
+// It contains labels for the warehouse domain
 // GENERATED CODE - DO NOT EDIT
 package labels
 
-// Labels for team warehouse
+import (
+	log "github.com/rudderlabs/rudder-go-kit/logger"
+)
+
 var (
-    UploadId = name[int]("uploadId")
-    UploadTime = name[Time]("uploadTime")
+	UploadID = func(v int64) log.Field { return log.NewIntField("uploadId", v) }
```

Comment:
[non-blocking] Should we introduce a prefix for domain-specific metrics ? or maybe put them in a different package ?

Reply:
We thought about it but avoided it to cause a conflict on purpose to make people rethink the label name. Perhaps if it's a conflict it might be a duplicate. If it's not a duplicate then a prefix can be added manually by changing the label name itself (i.e. by making the label name more verbose/explanatory for example).